src/libunicode/tables.rs

```diff
@@ -7603,13 +7603,13 @@ pub mod charwidth {
         }
     }
 
-    pub fn width(c: char, is_cjk: bool) -> Option<usize> {
+    pub fn width(c: char, is_cjk: bool) -> Option<u32> {
```

Comment:
I believe these files should be changed by changing `src/etc/unicode.py`, not by hand-editing.